code/workspaces/web-app/src/hooks/assetRepoHooks.js

```diff
@@ -0,0 +1,5 @@
+import useShallowSelector from './useShallowSelector';
+import assetRepoSelectors from '../selectors/assetRepoSelectors';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I find myself having to do this quite a lot and I wonder if it would be worth updating our eslint config to not have this as a rule?

Reply:
I think they generally make sense, if there's only one export why not make it default.  However it's worth overriding here to keep it consistent with the other hooks which may have multiple exports.  So I'm inclined to leave the rule as it is.  Very interesting discussion on the merits of the rule here: https://github.com/airbnb/javascript/issues/1365.

Reply:
That's a good point; it is probably good to just use this explicitly in the cases that is makes sense to do so!